tagstudio/src/qt/widgets/video_player.py

```diff
@@ -115,14 +118,21 @@ def __init__(self, driver: "QtDriver") -> None:
         autoplay_action.setCheckable(True)
         self.addAction(autoplay_action)
         autoplay_action.setChecked(
-            bool(self.driver.settings.value(SettingItems.AUTOPLAY, defaultValue=True, type=bool))
+            self.driver.settings.value(SettingItems.AUTOPLAY, defaultValue=True, type=bool)  # type: ignore
         )
         autoplay_action.triggered.connect(lambda: self.toggle_autoplay())
         self.autoplay = autoplay_action
 
         open_file_action = QAction("Open file", self)
         open_file_action.triggered.connect(self.opener.open_file)
-        open_explorer_action = QAction("Open file in explorer", self)
+
+        system = platform.system()
+        open_explorer_action = QAction("Open in explorer", self)  # Default, usually Linux
+        if system == "Darwin":
+            open_explorer_action = QAction("Reveal in Finder", self)
+        elif system == "Windows":
+            open_explorer_action = QAction("Open in Explorer", self)
```

Comment:
❓ this code already exists in `preview_panel.py`, cant it be reused here somehow?